inject/inject.vala

```diff
@@ -261,13 +284,19 @@ namespace Frida.Inject {
 			load_in_progress = true;
 
 			try {
-				var name = Path.get_basename (script_path).split (".", 2)[0];
-
+				string name;
 				string source;
-				try {
-					FileUtils.get_contents (script_path, out source);
-				} catch (FileError e) {
-					throw new Error.INVALID_ARGUMENT (e.message);
+
```

Comment:
Let's drop this blank line so it's clearer that the following logic is responsible for setting up these two variables.